includes/admin/class-wcmp-export.php

```diff
@@ -357,6 +360,16 @@ public function add_shipments(array $order_ids, bool $process)
             WCMP_Log::add("Shipment data for order {$order_id}.");
         }
 
+        if ($this->errors) {
+            setcookie('myparcel_response', implode('<br/>', $this->errors), time() + self::COOKIE_EXPIRE_TIME, "/");
+        }
+
+        if (0 === count($collection)) {
```

Comment:
Voor deze `0` zou ik een constante maken, zodat het duidelijker is waarom hij er staat

Reply:
Gedaan